client/client-multi/client-cpp/impl/channel/connectivity/IPConnectivityChecker.cpp

```diff
@@ -53,16 +53,9 @@ bool IPConnectivityChecker::checkConnectivity()
 
             boost::asio::ip::tcp::socket sock_(io_service);
             boost::asio::connect(sock_, ep);
-
-            KAA_LOG_INFO("Connection to the network exists");
-        } else {
-            KAA_LOG_WARN(boost::format("Unsupported ping server data (server=%p)") % server.get());
         }
-
         return true;
-    } catch (std::exception& e) {
-        KAA_LOG_INFO(boost::format("Connection to the network has disappeared: %1%") % e.what());
-    }
+    } catch (std::exception& e) {}
```

Comment:
Empty exception handler is not recommended. Any exceptional situation must be reported.